libs/testdiff/replacement.go

```diff
@@ -137,28 +138,31 @@ func (r *ReplacementsContext) SetPathWithParents(old, new string) {
 }
 
 func PrepareReplacementsWorkspaceClient(t testutil.TestingT, r *ReplacementsContext, w *databricks.WorkspaceClient) {
+	t.Helper()
+	PrepareReplacementsWorkspaceConfig(t, r, w.Config)
```

Comment:
I agree, we don't need to pass *databricks.WorkspaceClient if we always use w.Config and we should pass config directly.

In that case, we should completely remove PrepareReplacementsWorkspaceClient, it's a wrapper that is too thin.

optional, but would be more readable to to send this refactoring as separate PR and stack this on top.

Reply:
Yeah I agree with the removal, and that it should be a separate PR. I'll send a followup PR for this once this one is merged.